.npmrc

```diff
@@ -1 +1,2 @@
 package-lock = false
+@jsr:registry=https://npm.jsr.io
```

Comment:
Would end users need to add this to their `.npmrc` files?

Reply:
I've tried using `npm pack`-ed version of config-array from this branch in eslint/eslint, and `npm install` fails with:

```
$ npm i
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@jsr%2fstd__path - Not found
npm ERR! 404
npm ERR! 404  '@jsr/std__path@^1.0.2' is not in this registry.
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

Reply:
I don't think we can expect users to change their npm settings so that they can download packages from JSR. We could distribute `@std/path` in a separate directory inside `config-array` and import it from there. Or we could use the URL of the tarball from JSR to specify the dependecy in package.json:

```diff
-    "@jsr/std__path": "^1.0.2",
+    "@jsr/std__path": "https://npm.jsr.io/~/11/@jsr/std__path/1.0.2.tgz",
```

Either way, we will lose the magic of the caret `^` to pick newer versions automatically. I checked the JSR docs at https://jsr.io/docs/using-packages but I didn't find any recommendations on how to use their packages with npm apart from registering JSR in .npmrc.

Reply:
Can one of you open an issue on JSR for this? I can't imagine they didn't encounter this before, but before we spin our wheels, it would be nice to verify if we're doing things the right way.